vm-systemd/qubes-qrexec-agent.service

```diff
@@ -1,6 +1,6 @@
 [Unit]
 Description=Qubes remote exec agent
-After=qubes-dvm.service
+After=qubes-dvm.service xendriverdomain.service
```

Comment:
Wait a second though, if your concern is delayed startup: xendriverdomain.service does not need anything besides what qubes-qrexec-agent.service already depends on. It's literally just the daemon forking off. (I could make it even slimmer by writing a tiny native systemd unit instead to replace that gross rc.d script.) So all the code from https://github.com/marmarek/qubes-vmm-xen/commit/8639b6986ff15aed18c5f47ec56328b4a66f5763 could be avoided?